test/unit/acl/access-controller.test.ts

```diff
@@ -0,0 +1,31 @@
+jest.mock('rdflib')
+// import * as RdfLib from 'rdflib'
+jest.mock('solid-auth-client')
+// import * as SolidAuthClient from 'solid-auth-client'
+// import { DataBrowserContext } from 'pane-registry'
+const AccessController = require('../../../src/acl/access-controller')
+// just a note that the only public functions are isEditable, render, and save
+describe('AccessController', () => {
+  debugger
+  it('exists', () => {
+    expect(AccessController).toBeTruthy()
+  })
+  /* leaving this for now
```

Comment:
Maybe specify why it's left in so future readers know whether they can remove it.

Reply:
Ah sorry, fixed